pkg/agent/types/flags.go

```diff
@@ -181,6 +181,10 @@ type Flags interface {
 	SetLogIngestServerURL(url string) error
 	LogIngestServerURL() string
 
+	// LogShippingLevel is the level at which logs should be shipped to the server
+	SetLogShippingLevel(level string) error
+	LogShippingLevel() string
```

Comment:
Flags implies command line / config presence, right? I'm not sure we need to expose `LogShippingLevel` at that level. Default it to Error, and allow the control server to override.

Reply:
(eg: can logshipper be a subscriber of the control system)

Reply:
Okay! I still haven't totally absorbed how the flags worked out